api/envoy/config/core/v3/protocol.proto

```diff
@@ -53,8 +53,37 @@ message QuicKeepAliveSettings {
   }];
 }
 
+// Config for QUIC connection migration across network interfaces, i.e. cellular to WIFI, upon
+// network change events from the platform, i.e. the current network gets
+// disconnected, or upon the QUIC detecting a bad connection. After migration, the
+// connection may be on a different network other than the default network
+// picked by the platform. A connection is this special state is only allowed to
+// serve new requests for a certain period of time before being drained, and
+// meanwhile, QUIC will try to migrate to the default network if possible.
+message ConnectionMigrationSettings {
+  // Whether to migrate idle connections which aren't serving any requests.
+  // If false, idle connections will not be migrated but be closed upon
+  // migration signals.
+  bool migrate_idle_connections = 1;
+
+  // If idle connections are allowed to be migrated, only migrate the connection
+  // if it hasn't been idle for longer than this idle period. Otherwise, the
```

Comment:
Just top make sure I understand correctly, this implies that once a migration signal was received, the implementation will look "backwards" at all the connections and see if any of them has been idle (according to the set timeout), and if they are not, they will be migrated. Is that correct?
(I just want to make sure that this doesn't imply that once a migration signal arrives, a timer starts for all the connections to see which is idle and which is not).